Clients/src/presentation/components/ProjectCard/index.tsx

```diff
@@ -0,0 +1,72 @@
+import { Typography, Box } from "@mui/material";
+import Button from '@mui/material/Button';
+import { FC } from "react";
+import euimg from "../../assets/imgs/eu-ai-act.jpg"
+import ProgressBar from "./ProgressBar";
+
+export interface ProjectCardProps {
+    id: number,
+    name: string,
+    owner_name: string,
+    last_update: string,
+    controls_completed: string,
+    requirements_completed: string
+}
+
+const ProjectCard: FC<ProjectCardProps> = ({
+    name,
+    owner_name,
+    last_update,
+    controls_completed,
+    requirements_completed
+}) => {
+    const proggressCount = (progressString: string): number => {
```

Comment:
I found a very funny that! I don't know why, but this week we're making lots of mistakes in writing English words 😂😂😂😂😂
Almost every pull request I check has some spelling mistakes 🤣🤣
Please change this one `progressCount` my friend ✌️🙏🏻